src/Ellipsoid.jl

```diff
@@ -129,3 +129,23 @@ function σ(d::AbstractVector{N},
     α = sqrt(dot(d, E.shape_matrix * d))
     return E.center .+ E.shape_matrix * d ./ α
 end
+
+"""
+    ∈(x::AbstractVector{N}, E::Ellipsoid{N})::Bool where {N<:AbstractFloat}
+
+Check whether a given point is contained in an ellipsoid.
+
+### Input
+
+- `x` -- point/vector
+- `E` -- ellipsoid
+
+### Output
+
+`true` iff `x ∈ E`.
+"""
```

Comment:
Is there a way to explain the algorithm other than just repeating the code?

Reply:
i added the math